foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-CorpLzPolicySetDefinition.json

```diff
@@ -10,7 +10,133 @@
+                                            "where": {
+                                                "field": "Microsoft.Network/privateEndpoints/privateLinkServiceConnections[*].groupIds[*]",
+                                                "equals": "sqlServer"
+                                            }
+                                        },
+                                        "greaterOrEquals": 1
+                                    }
+                                ]
+                            },
+                            "then": {
+                                "effect": "[[parameters('effect')]",
+                                "details": {
+                                    "type": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups",
+                                    "roleDefinitionIds": [
+                                        "/providers/Microsoft.Authorization/roleDefinitions/4d97b98b-1d4f-4787-a291-c67834d212e7"
+                                    ],
+                                    "existenceCondition": {
+                                        "allOf": [
+                                            {
+                                                "field": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups/privateDnsZoneConfigs[*].privateDnsZoneId",
+                                                "equals": "[parameters('privateDnsZoneId')]"
+                                            }
+                                        ]
+                                    },
+                                    "deployment": {
+                                        "properties": {
+                                            "mode": "incremental",
+                                            "template": {
+                                                "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                                                "contentVersion": "1.0.0.0",
+                                                "parameters": {
+                                                    "privateDnsZoneId": {
+                                                        "type": "string"
+                                                    },
+                                                    "privateEndpointName": {
+                                                        "type": "string"
+                                                    },
+                                                    "location": {
+                                                        "type": "string"
+                                                    }
+                                                },
+                                                "resources": [
+                                                    {
+                                                        "name": "[concat(parameters('privateEndpointName'), '/deployedByPolicy')]",
```

Comment:
Please escape the ARM template functions properly using this pattern "[[parameter('paramName')]" if not, it is not deployable.